script/run_bench.py

```diff
@@ -11,8 +11,8 @@
 import sys
 
 # Runs the tests.
-WREN_DIR = os.path.dirname(os.path.dirname(os.path.realpath(__file__)))
-BENCHMARK_DIR = os.path.join(WREN_DIR, 'benchmark')
+HOME_DIR = os.path.dirname(os.path.dirname(os.path.realpath(__file__)))
```

Comment:
It mostly depend on what you're calling "home". :)

In my opinion, "home" is the place where all the Wren source-and-binary stuff resides. I'm open to suggestions, however.